app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.ui.example.login;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+
+/**
+ * A simple {@link WolmoFragment} subclass.
+ */
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+
+    @Override
+    public void init() {
```

Comment:
If init class will be empty, maybe we can leave it like init() {}

Reply:
✔ **Done**